python/publish_transforms_to_s3.py

```diff
@@ -0,0 +1,38 @@
+"""
+Python Script to Publish all Transforms (for every DW) to S3
+"""
+import json
+import rasgotransforms
+import boto3
```

Comment:
```suggestion
import json

import boto3

import rasgotransforms
```

Standard lib 

External 

Model